### PR TITLE
Henry Schmid

### PR DESCRIPTION
**Pull request by hschmid516**
Henry Schmid's party_bus_2107 Independent Challenge

### CODE REVIEW

File: lib/bus.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class Bus
+attr_reader :name, :capacity, :passengers, :number_of_passengers
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @passengers = []
+    @number_of_passengers = 0
```
Comment (mikedao) [inline lib/bus.rb:8]: We dont want to store number of passengers, it should be calculated from the passengers array

File: lib/bus.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  end
+
+  def yell_at_passengers
+    @passengers.map do |passenger| passenger.upcase
+    end
+  end
+
+  def over_capacity?
+    return true if @number_of_passengers > 4
+    false
```
Comment (mikedao) [inline lib/bus.rb:23]: This does not cover general cases. 

We can shorten this to `@number_of_passengers > @capacity`

**Review by mikedao — COMMENTED** [review]
Overall good work

Score: Pass

File: lib/bus.rb (hunk 1 above)
Comment (hschmid516) [reply lib/bus.rb:8]: Thanks Mike! I tried this originally, but it wasn't working for some reason so I ended up finding a workaround. I just figured it out and pushed the updates to my repository :)
